Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11643.xaml.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+using System.Collections.Generic;
+using System;
+
+#if UITEST
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.Brush)]
+#endif
+#if APP
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 11643, "[Bug] SwipeBehaviorOnInvoked=RemainOpen triggers event twice",
+		PlatformAffected.Android | PlatformAffected.iOS)]
+	public partial class Issue11643 : TestContentPage
+	{
+		public Issue11643()
+		{
+#if APP
+			Title = "Issue 11643";
+			Device.SetFlags(new List<string> { ExperimentalFlags.SwipeViewExperimental });
+			InitializeComponent();
+#endif
+		}
+
+		protected override void Init()
+		{
+
+		}
+
+#if APP
```

Comment:
Can we automate SwipeItems? If so, this test should be automated. If not, we need some instructions for whoever is manually running it (e.g., "Open the Swipeitem; if the alert displays twice, this test has failed".)